atc/api/jobserver/badge.go

```diff
@@ -4,23 +4,25 @@ import (
 	"bytes"
 	"fmt"
 	"net/http"
+	"net/url"
 	"text/template"
 
 	"github.com/concourse/concourse/atc/db"
 )
 
 var (
-	badgePassing = Badge{width: 88, fillColor: `#44cc11`, status: `passing`}
-	badgeFailing = Badge{width: 80, fillColor: `#e05d44`, status: `failing`}
-	badgeUnknown = Badge{width: 98, fillColor: `#9f9f9f`, status: `unknown`}
-	badgeAborted = Badge{width: 90, fillColor: `#8f4b2d`, status: `aborted`}
-	badgeErrored = Badge{width: 88, fillColor: `#fe7d37`, status: `errored`}
+	badgePassing = Badge{width: 88, fillColor: `#44cc11`, status: `passing`, title: `build`}
+	badgeFailing = Badge{width: 80, fillColor: `#e05d44`, status: `failing`, title: `build`}
+	badgeUnknown = Badge{width: 98, fillColor: `#9f9f9f`, status: `unknown`, title: `build`}
+	badgeAborted = Badge{width: 90, fillColor: `#8f4b2d`, status: `aborted`, title: `build`}
+	badgeErrored = Badge{width: 88, fillColor: `#fe7d37`, status: `errored`, title: `build`}
 )
 
 type Badge struct {
```

Comment:
Can we eliminate `badgeTemplateConfig` and just use this type directly? If we just make its fields and methods public (`.statusWidth`, `.statusTextWidth`) the template should "Just Work".

Reply:
I'll look into it! Seems very reasonable